packages/models/src/huggingface/text-generation.ts

```diff
@@ -0,0 +1,210 @@
+import { HttpError, POST } from '@axflow/models/shared';
```

Comment:
We need to make a call on naming and be consistent. In the file path we have 'huggingface' but in the types and objects we have 'Hf'. We will also need to specify this in the package subpath export which has implications on the import path. I don't care if we use "hf" or "huggingface" but we should choose one and use it consistently everywhere. (maybe hf cause it's shorter?)